finalPrep/annotations.js

```diff
@@ -2,41 +2,71 @@
 // Use the double // to create a new comment
 
 //Build a Bear
```

Comment:
Excellent annotations! Great use of technical vocabulary. Everything is clear and accurate.